knowledge-base/snyk/actions/action-security.yml

```diff
@@ -0,0 +1,6 @@
+name: 'Snyk Docker'
```

Comment:
@Devils-Knight please update the file path to `snyk/actions/docker/action-security.yml`